### PR TITLE
Experiment Template

### LINKED ISSUE #15: Draft a template for research experiments
## Description

Write a markdown template (or other relevant format) that includes the main sections for what should be documented for each experiment. Essentials might include:

1. Description of changes made (eg. quantisation change, new loss function, new architecture, new model/benchmark)
2. Commit hash/branch
3. Tensorboard plots (i.e. train/valid loss curves)
4. Table of metrics
5. Reviewer

Will probably contain some overlap with #12 since we can put the metric in the modelcard.md on hugging face

## Checklist before resolving issue
- [ ] I have included the issue number in my PR description
- [ ] I have rebased/pulled in changes from `main` and fixed any conflicts
- [ ] If I have changed the models/training code, I have run `dry_run.sh` on a CUDA device successfully

### PR DESCRIPTION
**Pull request by BrownianNotion**
## Description  
  

Closes #15 

## Pre-Merge Checklist  
- [x] I have included the issue number in my PR description  
- [x] I have rebased/pulled in changes from `main` and fixed any conflicts  
- [x] If I have changed the models/training code, I have run `dry_run.sh` on a CUDA device successfully

### CODE REVIEW

**Comment by BrownianNotion** [comment]
We can amend this later if needed, will merge so that we can use it now.
